packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -158,4 +175,34 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
     await enableUsageStats(billingLeadersIds, orgId)
     await sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader)
   }
+
+  await scheduleJobs(scheduledLockAt, orgId)
+}
+
+export const processLockOrganizationJob = async (
+  job: ScheduledJobOrganizationLock,
+  {dataLoader}: {dataLoader: DataLoaderWorker}
+) => {
+  const {orgId, runAt} = job
+
+  const organization = await dataLoader.get('organizations').load(orgId)
+
+  // Skip the job if unlocked or already locked or scheduled lock date changed
+  if (
+    !organization.scheduledLockAt ||
+    organization.lockedAt ||
+    organization.scheduledLockAt.getTime() !== runAt.getTime()
```

Comment:
-1 here we're returning if the `runAt` timestamp is different to the `scheduledLockAt` timestamp, but as `runScheduledJobs` happens every 10 minutes, it's likely that the timestamps will never match. 

I think we need to check if the date is the same rather than the timestamp

Reply:
@nickoferrall We set `runAt` value at the job creation, so it is not actual real-time value, but the value when job is supposed to be run.

https://github.com/ParabolInc/parabol/pull/7606/files#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR46

So If `runAt !== scheduledLockAt` that means that lock time was changed for some reason, and we don't need to run this job.

Reply:
OK I see, that makes sense 👍